src/components/player/PlayerBar.tsx

```diff
@@ -198,6 +199,13 @@ const PlayerBar = () => {
     setCurrentTime
   );
 
+  // Handle mpris volume in/out
+  useEffect(() => {
+    const volume = Number(playQueue.volume.toPrecision(2));
+    setLocalVolume(volume);
+    ipcRenderer.send('volume', volume);
+  }, [playQueue.volume]);
+
```

Comment:
I'm thinking it might be better to put this in the `usePlayerControls` hook since all the MPRIS controls are currently located there.

Reply:
Do you want to me match your implemented of the other MPRIS controls (getting an event from the main process)? Or do you just want to move the function over to `usePlayerControls`?

Reply:
I think it would actually make sense to move the `ipcRenderer.send('volume', volume)` within the volume handlers themselves `handleVolumeSlider`, `handleVolumeKey`, and `handleVolumeWheel`.

Reply:
Alright, I moved the MPRIS updates into the volume handlers, which like you said makes more sense :)
I left the MPRIS handler where it was and put an update into the handler for the volume slider inside PlayerBar.tsx so that MPRIS gets the volume at startup.

Reply:
Thanks. I'll test it out later today (have to reboot into Linux) and then merge if I don't find any issues.